packages/adblocker-puppeteer/adblocker.ts

```diff
@@ -310,7 +311,7 @@ export class PuppeteerBlocker extends FiltersEngine {
           contentType: redirect.contentType,
         });
       }
-    } else if (match === true) {
```

Comment:
This change (and similar changes in this file) does not seem related to fixing the listener issue.

Reply:
Sorry, it was an IDE's automatic refactoring :D
Gonna roll it back.